packages/app-desktop/gui/NoteTextViewer.tsx

```diff
@@ -132,9 +131,14 @@ export default class NoteTextViewerComponent extends React.Component<Props, any>
 	public focus() {
 		if (this.webviewRef_.current) {
 			focus('NoteTextViewer::focus', this.webviewRef_.current);
+			this.send('focus');
```

Comment:
Previously, after `.focus`ing the viewer, the arrow keys didn't scroll the viewer's content.

Reply:
Could you this comment in the code? I'm not a big fan of having two different ways to focus something but if it's an exception we can leave it but with a comment.